ListingService/Sources/SubredditSections.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
```

Comment:
Are we going to use this enum again, is a separate VC / datasource? 

If not, I wonder if it would be better suited, placed within `SubredditsViewController.swift` perhaps in the ` extension SubredditsViewController: UITableViewDataSource, UITableViewDelegate ` as its related?